source/release-notes/backend.txt

```diff
@@ -13,6 +13,15 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240807:
+
+07 August 2024 Release
+~~~~~~~~~~~~~~~~~~~~~~
+
+- Fixed bug where some destructive changes in push-pull were not being handled properly
+- Fixed bug where unlinking a datasource with Sync enabled would crash the UI
```

Comment:
When we talk about data sources in the docs, we tend not to concatenate the term. i.e. in the ToC, we have: "Connect to MongoDB Data Sources".

```suggestion
- Fixed bug where unlinking a data source with Sync enabled would crash the UI
```